### PR TITLE
Updated khan-exercises directory paths

### PR DESCRIPTION
**Pull request by Antrikshy**
Fixes issue #2902 and possibly various other problems.

### CODE REVIEW

File: kalite/distributed/settings.py [hunk 1]
```diff
@@ -98,7 +98,7 @@ def USER_FACING_PORT():
 CENTRAL_SERVER_HOST   = getattr(local_settings, "CENTRAL_SERVER_HOST",   ("adhoc.%s:7007" if DEBUG else "kalite.%s") % CENTRAL_SERVER_DOMAIN)
 CENTRAL_WIKI_URL      = getattr(local_settings, "CENTRAL_WIKI_URL",      "http://kalitewiki.%s/" % CENTRAL_SERVER_DOMAIN)
 
-KHAN_EXERCISES_DIRPATH = os.path.join(os.path.dirname(__file__), "static", "khan-exercises")
+KHAN_EXERCISES_DIRPATH = os.path.join(os.path.dirname(__file__), "static", "perseus", "ke")
```
Comment (aronasorman) [inline kalite/distributed/settings.py]: Actually, we'll wanna move this definition out of here and into kalite/settings.py.

File: kalite/topic_tools/__init__.py [hunk 2]
```diff
@@ -430,7 +430,7 @@ def get_exercise_data(request, exercise_id=None):
     if not exercise:
         return None
 
-    exercise_root = os.path.join(settings.KHAN_EXERCISES_DIRPATH, "exercises")
+    exercise_root = os.path.join(settings.KHAN_EXERCISES_DIRPATH)
```
Comment (rtibbles) [inline kalite/topic_tools/__init__.py]: I don't believe this edit is necessary. It should be looking inside the exercises folder.

File: kalite/settings.py [hunk 3]
```diff
@@ -170,7 +170,6 @@ def package_selected(package_name):
 import_installed_app_settings(INSTALLED_APPS, globals())
 
 # Override
-KHAN_EXERCISES_DIRPATH = getattr(local_settings, "KHAN_EXERCISES_DIRPATH", os.path.join(STATIC_ROOT, "khan-exercises"))
```
Comment (aronasorman) [inline kalite/settings.py]: Move the definition from distributed/settings.py, then wrap it in the `getattr(local_settings, "KHAN_EXERCISES_DIRPATH", ...)` call, to allow users to override it.

File: kalite/topic_tools/__init__.py (hunk 2 above)
Comment (aronasorman) [inline kalite/topic_tools/__init__.py]: Breaks i18n. Will fix in a later PR.

Comment (rtibbles) [inline kalite/topic_tools/__init__.py]: How? The edit to i18n above will make language pack exercise templates unpack into static/perseus/ke/exercises/{ka_lang_code} folder

**Comment by rtibbles** [comment]
Merging to iterate.

**Pull request merged**